docs/cudos-intercloud/usage/via-user-interface/private-networks.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Private Networks
+id: private-networks
+---
+
+# Private Networks
+
+You can improve the security of your VMs by not exposing them to the internet, but deploying them in a private network instead.
+As an added bonus, you will also save the IP address cost for each machine that's not directly exposed publicly.
+Thus, if your setup permits this can lead to significant savings when deploying many machines.
+
+## Creating a Private Network
+Creating a private network with CUDOS Intercloud is very simple.
+
+1. Navigate to the [`NETWORKS` page](https://intercloud.cudos.org/#/networks) in the top navigation menu.
+2. Click on `Create  a network`.
+3. Choose the details of your private network.
+    - **Location**: private networks are not shared between datacentres, so you need to choose in which datacentre you want to create your private network.
+    - **Name**: a unique identifier for your private network.
+    - **Subnet**: the address range for the private network. If you are unsure about what should go in here, you can learn more for example [here](https://www.ripe.net/about-us/press-centre/understanding-ip-addressing/).
+
+:::note
+CUDOS Intercloud's suppliers all use IPv4 at the moment.
+:::
+
+After that, your network will be created and you will be able to choose it in the `Networking` box of the VM creation page.
+
+## Creating a VM in a Private Network
+The process is the same as for any other VM, expcet that you need to select the private network that you want in the dropdown menu of the Networking box.
+Optionally, you can assign a public IP address, so that it can be reached from the outside as well.
+
+Please note that **you will always need at least one machine in your private network with a public IP address**, to act as a jump host.
+You need to connect via SSH to all machines remotely, so you need one that is going to act as the point of access to your network.
+It is recommended that this jump host is dedicated to this purpose, though it's not essential.
+
+## Connecting to a VM in a Private Network
```

Comment:
I would note that this is only one way to use a jump host. It isn't _necessary_ to use the SSH jump host feature, even if it is the most convenient method. You can also SSH into it normally and then SSH again _from_ there. Or even proxy/tunnel some other protocol entirely.

So I might make it clearer that this the "SSH jump host" approach to remote access, which happens to be recommended, but it's just one of the approaches.